src/main/java/frc/robot/subsystems/ShooterHoodSubsystem.java

```diff
@@ -8,27 +8,29 @@
 import com.revrobotics.CANPIDController;
 import com.revrobotics.CANDigitalInput;
 
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 import frc.robot.Constants;
 import frc.robot.utils.MoPrefs;
-import frc.robot.utils.SimmableCANSparkMax;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class ShooterHoodSubsystem extends SubsystemBase {
-  private final CANSparkMax hoodNEO = new SimmableCANSparkMax(Constants.SPARKMAX_SHOOTER_HOOD_CAN_ADDR,
+  private final CANSparkMax hoodNEO = new CANSparkMax(Constants.SPARKMAX_SHOOTER_HOOD_CAN_ADDR,
```

Comment:
What did I miss? The SimmableCANSparkMax should have been a seamless replacement